src/apps/aave/positions.ts

```diff
@@ -121,7 +121,30 @@ const hook: PositionsHook = {
               imageUrl: AAVE_LOGO,
             },
             dataProps: {
-              apy: supplyApy,
+              // TODO(ACT-1328): Add manageUrl and contractCreatedAt
+              yieldRates: [
+                {
+                  yieldRatePercentage: supplyApy,
+                  label: 'earnFlow.yieldRateLabels.earningsApy',
+                  tokenId: getTokenId({
+                    networkId,
+                    address: reserveData.underlyingAsset.toLowerCase(),
+                  }),
+                },
+              ],
+              earningItems: aTokenRewardsInfo?.length
+                ? aTokenRewardsInfo.map((info) => ({
+                    amount: toDecimalNumber(
+                      info.userUnclaimedRewards,
+                      info.rewardTokenDecimals,
+                    ),
+                    label: 'earnFlow.earningItemLabels.rewards',
```

Comment:
This seems tightly coupled with the wallet, wonder if we should define some kind of enum, and then the wallet can map this to an appropriate string. Although that would mean there would be BC issues if we want to add new types. @jeanregisser what do you think?

Reply:
Yes I think eventually the hooks repo/service should be localized.
I left a comment about this in https://linear.app/valora/issue/ACT-1329/add-earnings-hooks-translation-strings-to-wallet#comment-8c01cfac